src/test/java/org/apache/cassandra/sidecar/server/ServerSSLTest.java

```diff
@@ -124,6 +123,7 @@ void tearDown() throws InterruptedException
     void failsWhenKeyStoreIsNotConfigured()
     {
         builder.sslConfiguration(SslConfigurationImpl.builder().enabled(true).build());
+        vertx = vertx();
```

Comment:
why is this change needed?

Reply:
Since now creating a Vert.x instance needs `SidecarConfiguration` to get metrics options. Changed this. Because in these tests, the configuration is updated inside test as per each test. After the configuration update, building the vert.x instance

Reply:
It is not needed for all tests cases, example some of the failure test cases. But changed it consistently for easier test reading